Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionCrowdStrikeFalcon.yaml

```diff
@@ -0,0 +1,296 @@
+            | extend
+                EventCount = matchCount,
+                EventStartTime = unixtime_milliseconds_todatetime(tolong(ReceiptTime)),
+                NetworkDirection = case(
+                        connectionDirection == 1, "Inbound",
+                        connectionDirection == 2, "Outbound",
+                        ""
+                    ),
+                SrcIpAddr = case(
+                    connectionDirection == 1, remoteAddress,
+                    connectionDirection == 2, localAddress,
+                    ""
+                ),
+                SrcPortNumber = case(
+                        connectionDirection == 1, remotePort,
+                        connectionDirection == 2, localPort,
+                        int(null)
+                    ),
+                DstIpAddr = case(
+                    connectionDirection == 1, remoteAddress,
+                    connectionDirection == 2, localAddress,
+                    ""
+                ),
+                DstPortNumber = case(
+                        connectionDirection == 1, localPort,
+                        connectionDirection == 2, remotePort,
+                        int(null)
+                    ),
+                deviceIp = iff(hostName matches regex "(([0-9]{1,3})\\.([0-9]{1,3})\\.([0-9]{1,3})\\.(([0-9]{1,3})))", hostName, "")
+            | extend 
+                hostName = iff(isempty(deviceIp), hostName, ""),
+                AdditionalFields = bag_pack(
+                      "networkProfile", networkProfile,
+                      "ruleDescription", ruleDescription,
+                      "ruleGroupName", ruleGroupName,
+                      "cmdLine", cmdLine
+                  ),
+                NetworkIcmpCode = icmpCode
+            | invoke _ASIM_ResolveDvcFQDN('hostName')
+            | invoke _ASIM_ResolveNetworkProtocol('protocol')
+            | extend NetworkIcmpType = _ASIM_LookupICMPType('icmpType')
+            | lookup EventFieldsLookup on ruleAction
+            | project-rename
+                DvcId = deviceId,
+                DvcIpAddr = deviceIp,
+                EventOriginalSubType = eventType,
+                NetworkRuleName = ruleName
+            | extend
+                Rule = NetworkRuleName,
+                Dvc = coalesce(DvcId, DvcHostname, DvcIpAddr);
+        let networkaccessdata = alldata
+            | where DeviceEventClassID has "Network Access In A Detection Summary Event"
+            | parse-kv AdditionalExtensions as (CSMTRPatternDisposition: string) with (pair_delimiter=";", kv_delimiter="=")
```

Comment:
@niralishah-crest @jayeshprajapaticrest 
the logs I have had following fields in additionalextensions. Can you please extract these fields and then use either these extracted values or Activity, DeviceAction values.

Reply:
tactic=Malware;technique=PUP;objective=Falcon Detection Method;patternDisposition=Detection/Quarantine, standard detection and quarantine was attempted.

Reply:
We don't have such logs where this type of fields are available in AdditionalExtensions. But We have added a condition as you have given the example.